src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -50,6 +50,14 @@ provideEditingConfig(Homepage, {
       description: 'Needs to be approved at https://cards-dev.x.com/validator',
     },
     siteUserProfilePage: { title: 'Location of user profile page' },
+    siteFontBodyWeight: {
+      title: 'Body weight',
+      description: 'Default: 500',
+    },
+    siteFontHeadlineWeight: {
+      title: 'Headline weight',
+      description: 'Default: 500',
+    },
```

Comment:
I read this text and didn't understand it. The plan is that you can change fonts. The main use case is to prepare better sales demos to achieve a more realistic CI of potential customers.

So you also need to customize the font and weight. If you don't give the option to customize the weight, you will always only deliver a normal weight. In many cases, you will not be able to create a website that resembles the customer's CI (typographically speaking).

Here an example of Viessmann CI. Headings and body text also differ in font-face and weight. So how can we leave this option out? Of course, this is a Pro feature. But the whole thing is only for Pro users. Standard Scrivito customers will never see this font option because an average editor is not allowed to change the fonts or at least this option has to be restricted.

![Bildschirmfoto 2024-12-12 um 11 06 35](https://github.com/user-attachments/assets/ddad68df-6bd8-4b87-bfe0-6eb96a294b76)

Reply:
Thanks for the example. This is exactly the kind of CI spec I have in mind.
The example CI specifies **4 font names**, together with font sizes and color.
* Univers 57 Condensed (https://fontsgeek.com/fonts/univers-57-condensed-regular)
* Univers 45 Light (https://fontsgeek.com/fonts/univers-45-light-regular)
* Produktfont "V"
* Univers 55 Roman (https://fontsgeek.com/fonts/univers-55-roman)

With the two font options we support, to make the website appear as expected the editor would upload the fonts for "Fließtext" (Univers 45 Light) and "Headline" (Univers 57 Condensed).
The font sizes currently can't be adjusted, but for a demo or prototype this is not needed, plus headlines have an array of sizes available for a good approximation. Same with font colors.

My feedback is about the configurable **font weight**. The CI doesn't specify any font weight. This makes sense, since each font comes with a default weight. Maybe the confusion comes from the fact that there is a font named "Univers 45 Light", which apparently is a font which comes with a low default weight. But as far as I understand this, this is not a user setting, but the default, just as the condensed nature of "Univers 57 Condensed" is not a user setting but a default property of the respective font.

Am I misinterpreting something here? Is there another CI example which makes the need for such a feature more obvious?

Reply:
Dave... Why do you do this? Th CI exactly defines font weight. There are different naming convention out there. 
e.g. 45 -> light -> 300

So you say we should decline the weight value because you don't see the use case of a font weight?

Reply:
I see the use of font weight in the design, e.g., `<b>` or `<strong>` displays with a higher weight.

I uploaded FiraSans-Bold-700.woff2, FiraSans-Regular-400.woff2, and FiraSans-Light-300.woff2, left the "Body weight" blank and set "Body font" to one of the font versions. In all three cases the body of the homepage showed in the respective font width, which matches my expectations and should be enough to allow for matching the company CI.

Putting a value in "Body weight" has strange effects though:
* Using FiraSans-Light-300.woff2 and setting the weight to 300 shows bold text incorrectly.
* Using FiraSans-Regular-400.woff2 and setting the weight to 300 shows bold text incorrectly.
* Using FiraSans-Bold-700.woff2 and setting the weight has no effect at all.
* Only when using all three fonts and specifying their weights setting the weight to 300 bold text is still shown correctly, but the same effect can be achieved by just using FiraSans-Light-300.woff2.

That's why this weight value is confusing to me and I don't understand how an editor is supposed to use it in the site fonts settings. I think that just uploading and selecting the desired width of a font will do regarding the CI.

If I'm still missing the point, maybe you could give an example where the weight setting is mandatory, so I can give some feedback about how to, e.g., improve the wording of the weight settings.

Reply:
For improved wording/UI see aeb03cd